example-mlem-get-started/generate.sh

```diff
@@ -0,0 +1,210 @@
+#!/bin/bash
+# See https://mlem.ai/doc/get-started
+
+# Setup script env:
+#   e   Exit immediately if a command exits with a non-zero exit status.
+#   u   Treat unset variables as an error when substituting.
+#   x   Print commands and their arguments as they are executed.
+set -eux
+
+while read var; do
+  [ -z "${!var+x}" ] && { echo "$var is empty or not set"; exit 1; }
+done << EOF
+AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY
+HEROKU_API_KEY
+AZURE_STORAGE_CONNECTION_STRING
+EOF
+
+HERE="$( cd "$(dirname "$0")" ; pwd -P )"
+REPO_NAME="example-mlem-get-started"
+
+BUILD_PATH="$HERE/build"
+
+mkdir -p $BUILD_PATH
+pushd $BUILD_PATH
+if [ ! -d "$BUILD_PATH/.venv" ]; then
+  virtualenv -p python3 .venv
+  export VIRTUAL_ENV_DISABLE_PROMPT=true
+  source .venv/bin/activate
+  echo '.venv/' > .gitignore
+  pip install -r $HERE/code/src/requirements.txt
+  git clone https://github.com/iterative/mlem.git
+  pip install -e ./mlem
+fi
+popd
+
+source $BUILD_PATH/.venv/bin/activate
+
+REPO_PATH="$HERE/build/$REPO_NAME"
+
+#if [ -d "$REPO_PATH" ]; then
+#  echo "Repo $REPO_PATH already exists, please remove it first."
+#  exit 1
+#fi
+
+TOTAL_TAGS=12
+STEP_TIME=100000
+BEGIN_TIME=$(( $(date +%s) - ( ${TOTAL_TAGS} * ${STEP_TIME}) ))
+export TAG_TIME=${BEGIN_TIME}
+export GIT_AUTHOR_DATE=${TAG_TIME}
+export GIT_COMMITTER_DATE=${TAG_TIME}
+tick(){
+  export TAG_TIME=$(( ${TAG_TIME} + ${STEP_TIME} ))
+  export GIT_AUTHOR_DATE=${TAG_TIME}
+  export GIT_COMMITTER_DATE=${TAG_TIME}
+}
+
+export GIT_AUTHOR_NAME="Mikhail Sveshnikov"
+export GIT_AUTHOR_EMAIL="mike0sv@gmail.com"
+export GIT_COMMITTER_NAME="$GIT_AUTHOR_NAME"
+export GIT_COMMITTER_EMAIL="$GIT_AUTHOR_EMAIL"
+
+
+
+mkdir -p $REPO_PATH
+pushd $REPO_PATH
+
+git init -b main
+cp $HERE/code/README.md .
+cp -r $HERE/code/.devcontainer .
+cp $HERE/.gitattributes .
+cp $HERE/code/src/requirements.txt .
+cp $HERE/code/src/*.py .
+echo ".venv" > .gitignore
+git add .
+tick
+git commit -m "Initialize Git repository"
+git tag -a "0-git-init" -m "Git initialized."
+
+
+################## MLEM
+
+git branch simple
```

Comment:
Q: so, do we use main or simple? do we use main at all?

Reply:
We use `main` later on L140 as a base for `dvc` branch. So `main` is the common beginning for both non-DVC and DVC cases.